File: application-restricted-api-key-tutorials/csharp/HelloWorld.cs
using System.Net;

namespace csharp;

internal static class HelloWorld
{
    public static async Task Main(String[] args)
    {
        
        var clientId = Environment.GetEnvironmentVariable("CLIENT_ID")!;
        var endpoint = Environment.GetEnvironmentVariable("ENDPOINT")!;

        var response = await SendRequest(endpoint, clientId);
        Console.WriteLine($"Response from Hello World API:\n{response}");
    }

    private static async Task<string> SendRequest(string url, string clientId)
    {
        Console.WriteLine($"Sending GET {url}");

        var client = new HttpClient();
        client.DefaultRequestHeaders.Add("apikey", $"{clientId}");
        
        var response = await client.GetAsync(url);

        return await response.Content.ReadAsStringAsync();
        
    }
}

File: application-restricted-api-key-tutorials/php/HelloWorld.php
<?php

$clientId = getenv("CLIENT_ID");
$endpoint = getenv("ENDPOINT");

$response = sendRequest($clientId, $endpoint);
echo "Response from Hello World API: " . $response . "\r\n";

function sendRequest($clientId, $endpoint)
{
  $curl = curl_init($endpoint);
  curl_setopt($curl, CURLOPT_URL, $endpoint);
  curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);

  $headers = array(
      "Accept: application/json",
      "apikey: $clientId",
  );
  curl_setopt($curl, CURLOPT_HTTPHEADER, $headers);
  $resp = curl_exec($curl);
  curl_close($curl);
  return $resp;
}
